### ISSUE TITLE
Latest block by Eme

### ISSUE THREAD

**Issue by Omnomberry** [lunyx/CasualMeter#155]
Any insight on the latest 3rd party block by Eme ?
If its possible to workaround on cm.
http://tera.enmasse.com/news/posts/08-24-17-server-maintenance-and-security-update

**Comment by Milice** [comment]
From my understanding, CasualMeter is discontinued, the rumours might be false but either way, i'll look in to this when i get home from work.

**Comment by lunyx** [comment]
The core functionality is shared with Shinra, so if Shinra works, this will also be updated to work.

**Comment by Thizzx3w** [comment]
would this be  reason why my casual meter is crashing every time it connects to server? "there is an unexpected error. please check log files for more detail"

**Comment by Omnomberry** [comment]
Yes, it started with this update.
tl/dr Eme tried to stop valks who cheat with a script that one shots bosses, this broke proxy and dps meter, but valks apparently still can one shot bosses  ...
The thread on Eme forum talking about this got deleted as far as I can see.

**Comment by Thizzx3w** [comment]
It is, used it yesterday, just downloaded the latest one. Someone else said some other meter didn't go down at all, he didn't say which. I just assumed shinra

**Comment by Omnomberry** [comment]
But Shinra forces win 10 creators update (for Net 4.7) :sob: 
Thats almost 2 years of updating for me u.u
Stupid Eme breaking CM for no benefit

**Comment by Gl0** [comment]
You can replace Tera.Core.dll with one taken from latest shinra as a temporary workaround.

**Comment by Omnomberry** [comment]
That makes it go past server select, but as soon as it gets dps data it crashes, do u need to copy more ?

**Comment by Gl0** [comment]
Not sure what's the problem in your case. Updated my [fork](https://github.com/Gl0/CasualMeter/releases) with latest tera.core and db, seems working (note that it doesn't have autoupdate included - so if you use it - you should update db manually).

**Comment by Omnomberry** [comment]
Thanks ! 
Your package works.
How often and how do you update the db ?

**Comment by Gl0** [comment]
Updated archive with fixed version of tera.core.dll, try redownload it.

**Comment by Omnomberry** [comment]
you mean v1.2.4.8B from yesterday ? Yea, Ive been trying that version ;w;

**Comment by Gl0** [comment]
no, just updated file now without changing version

**Comment by Omnomberry** [comment]
I tried again, same issue, works couple seconds, then crash.
Apparently a NET exception, maybe cuz they are using the new 4.7 NET ;w;

**Comment by Gl0** [comment]
updated it another one time, recheck

**Comment by Omnomberry** [comment]
It hasnt crashed yet ! Im scared to restart but its working great now !
is updating db hard << ?

**Comment by Gl0** [comment]
just copy tera.core.dll from updated version of shinra if it changed + replace /res folder content with updated https://github.com/neowutran/TeraDpsMeterData

**Comment by Omnomberry** [comment]
Thanks ! Did you modify tera.core.dll to make the error go away ?

**Comment by Gl0** [comment]
Yes. You can always download latest version of this dll from [shinra update site](https://neowutran.ovh/updates/ShinraMeterV/Tera.Core.dll.zip)

**Comment by Omnomberry** [comment]
Thanks.
I guess I should leave the issue open so other ppl can see the solution

**Comment by lunyx** [comment]
I just updated the release, it should work now without having manually updated dlls.
